wagtailio/navigation/templatetags/navigation_tags.py

```diff
@@ -0,0 +1,53 @@
+from django import template
+
+register = template.Library()
+
+
+@register.inclusion_tag("patterns/includes/menu_footer.html", takes_context=True)
+def footer_navigation(context):
+    try:
+        footer_navigation = context["settings"]["navigation"][
+            "NavigationSettings"
+        ].footer_navigation
+    except (KeyError, AttributeError):
+        footer_navigation = None
+
+    return {
+        "footer_navigation": footer_navigation,
+    }
+
+
+@register.inclusion_tag("patterns/includes/menu_main.html", takes_context=True)
+def main_navigation(context):
+    try:
+        main_navigation = context["settings"]["navigation"][
+            "NavigationSettings"
+        ].main_navigation
+    except (KeyError, AttributeError):
+        return {}
+
+    menus = []
+    for item in main_navigation.menu_items.all():
```

Comment:
If you do not have a main nav defined, this fails with `'NoneType' object has no attribute 'menu_items'`

Reply:
Thanks for this. Fixed in 5d11060